CHANGELOG.rst

```diff
@@ -8,6 +8,7 @@ Added:
 ^^^^^^
 - More accurate reporting of stimulation subprotocol start times.
 - Notifications to let users know when their customer account has reached their analysis limit, if applicable.
+- User password and username redacted from logs.
```

Comment:
This should be under fixed and say something about how it caught a case where sensitive info could still be logged